projects/mercury/src/metadata/common/LinkedDataEntityForm.js

```diff
@@ -73,6 +73,29 @@ export const LinkedDataEntityForm = ({
         return <MessageDisplay message={errorMessage} />;
     }
 
+    function getPropertyListItem(p) {
+        return (
+            <ListItem
+                key={p.key}
+                disableGutters
+                style={{display: 'block'}}
+            >
+                <LinkedDataProperty
+                    formEditable={editable}
+                    property={p}
+                    values={values[p.key]}
+                    validationErrors={validationErrors[p.key]}
+                    onAdd={editable ? (value) => onAdd(p, value) : () => {
+                    }}
```

Comment:
it is a strange formatting, my ide complains about it. Could you make it in one line, as it was before?
```suggestion
                    onAdd={editable ? (value) => onAdd(p, value) : () => {}}
```